app/models/miq_report/formatting.rb

```diff
@@ -191,6 +191,23 @@ def format_datetime(val, options)
     val.strftime(options[:format])
   end
 
+  def format_relative_date(val, options)
+    return val unless val.kind_of?(Time) || val.kind_of?(Date)
```

Comment:
```suggestion
    return val unless val.kind_of?(Time) || val.kind_of?(Date) || val.kind_of?(DateTime)
```

Reply:
I agree. it makes it more clear.

Interestingly:

```ruby
>DateTime.new.kind_of?(Time)
# => false

>DateTime.new.kind_of?(Date)
# => true
```

Tangent: Also, rails should probably be using `Time` and not `DateTime`, but that ship has sailed many eons ago.

Reply:
@Fryguy implemented the request changes.Thanks